libpod.conf

```diff
@@ -35,6 +35,9 @@ conmon_env_vars = [
 # CGroup Manager - valid values are "systemd" and "cgroupfs"
 cgroup_manager = "systemd"
 
+# Container init binary
+#init_path = "/usr/bin/catatonit"
```

Comment:
Should this be installed in /usr/bin?  Or should it be installed in /usr/sbin. or /usr/lib/libpod/ or /usr/libexec/libpod/

Would anyone ever execute catatonic to run on their own or would it always be used by a container engine?

Reply:
Maybe we should convert this to a []string like runtime path, so we can accommodate different install paths

Reply:
> Would anyone ever execute catatonic to run on their own or would it always be used by a container engine?

I just treat it like any other binary but I see where you're pointing with `/usr/libexcec/libpod`.

> Maybe we should convert this to a []string like runtime path, so we can accommodate different install paths

I am not a fan of that as it will always slow down things. Personally, I expect packagers to configure that in `libpod.conf`.

Reply:
Now defaulting to `/usr/libexec/libpod`. @mheon, I think that's again something for the changelog. I will mention that as well in the commit message.